os/arch/arm/src/armv7-r/Kconfig

```diff
@@ -39,6 +39,12 @@ config ARMV7R_DCACHE_WRITETHROUGH
 	default n
 	depends on ARMV7R_DCACHE
 
+config MPU_STACKGUARD
```

Comment:
Is this working only for armv7-r? If it is, let's add ARMV7R_ prefix.
```
ARMV7R_MPU_XXX
```

Reply:
Yes. As of now, This is applicable only for ARMV7-R.  We will modify the changes and re-post it